.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: c
+sudo: required
+env:
+  global:
+    # Travis limits maximum log size, we have to cut tests output 
```

Comment:
If we're forced to truncate the logs it would be good to make the build itself quieter so there are more lines available for the test results.

This could be accomplished by running the testing in tree by using `make --no-print-directory -s` and it would have the update of speeding up the build which will allow more time for testing.